tests/phpunit/integration/Modules/Sign_In_With_GoogleTest.php

```diff
@@ -201,8 +168,6 @@ public function test_render_signinwithgoogle__woocommerce_active() {
 		);
 
 		// Render the button in the WooCommerce form.
-		do_action( 'woocommerce_loaded' );
-		do_action( 'woocommerce_login_form_start' );
 		$woo_output = $this->capture_action( 'woocommerce_login_form_start' );
```

Comment:
As mentioned above, this action should be sufficient for determining if WC is active.